addons/better-img-uploads/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Better image uploads",
+  "description": "Automatically converts uploaded images into svg images to keep them from losing quality.",
+  "tags": ["editor", "beta"],
+  "credits": [
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://scratch.mit.edu/users/ErrorGamer2000/"
+    }
+  ],
```

Comment:
False

Reply:
wut

Reply:
oh i did it on the wrong line oops

i was referring to `enabledByDefault` which is currently true.